packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -122,7 +125,9 @@ function parseDocumentReference(
     language: findSlotValue("languageCode"),
     size: sizeValue ? parseInt(sizeValue) : undefined,
     title: findClassificationName(XDSDocumentEntryClassCode),
-    creation: findSlotValue("creationTime"),
+    creation: findSlotValue("creationTime")
+      ? dayjs(findSlotValue("creationTime")).toISOString()
+      : undefined,
```

Comment:
dayjs precautions

Reply:
ditto regarding date creation shared func

Reply:
this one isnt the same as the other one and only gets used once in this function so gonna skip this one